html/interaction/focus/the-autofocus-attribute/not-on-first-task.html

```diff
@@ -12,6 +12,7 @@
 
 <script>
 "use strict";
+setup({ single_test: true });
```

Comment:
I'd prefer `test(() => { ... })` wrapping here and for the other synchronous tests in this PR. (This is the first one.)

Reply:
I've made the change you requested. Many of these tests (and still more in the related pull requests) could be expressed with a single `async_test` by swapping out `done()` for `t.done()`. Do you think that's preferable, too?